providers/aws/config/config.go

```diff
@@ -84,6 +84,12 @@ var Config = plugin.Provider{
 					Default: "",
 					Desc:    "Override option for EBS scanning that tells it to not create the snapshot or volume",
 				},
+				{
+					Long:    "scope",
```

Comment:
i would expect to be able to specify this filter at the resource level, not the cli level

Reply:
I was thinking about that, but how? When inside the shell providing arguments to resource doesnt' seem possible at the moment
```
failed to compile: cannot call resource field with arguments yet
```

Reply:
i was thinking similar to how we init objects sometimes with a value, like what u did here for example: https://github.com/mondoohq/cnquery/pull/2753/files

Reply:
This doesn't seem to work because changing it does not invalidate the cache, leading to situations like this

```
cnquery> aws.waf(scope: "REGIONAL").acls
aws.waf.acls: [
  0: aws.waf.acl name="marius-test"
]
cnquery> aws.waf(scope: "CLOUDFRONT").acls
aws.waf.acls: [
  0: aws.waf.acl name="marius-test"
]
```

In the example above, I should not get the same result with a changed scope, but the previous cached result is returned.

Reply:
yaa. i think the only alternative i can think of is to do aws.waf.regional.acls and aws.waf.cloudfront.acls kinda thing, so that we have two different aws waf prefixes.

i think this solution works for now, we'll see how users respond